sdk/storage/azure-storage-queue/azure/storage/queue/queue_client.py

```diff
@@ -0,0 +1,768 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import functools
+from typing import (  # pylint: disable=unused-import
+    Union, Optional, Any, IO, Iterable, AnyStr, Dict, List, Tuple,
+    TYPE_CHECKING)
+try:
+    from urllib.parse import urlparse, quote, unquote
+except ImportError:
+    from urlparse import urlparse # type: ignore
+    from urllib2 import quote, unquote # type: ignore
+
+import six
+
+from ._shared.shared_access_signature import QueueSharedAccessSignature
+from ._shared.utils import (
+    StorageAccountHostsMixin,
+    add_metadata_headers,
+    process_storage_error,
+    return_response_headers,
+    return_headers_and_deserialized,
+    parse_query,
+    serialize_iso,
+    parse_connection_str
+)
+from ._queue_utils import (
+    TextXMLEncodePolicy,
+    TextXMLDecodePolicy,
+    deserialize_queue_properties,
+    deserialize_queue_creation)
+from ._generated import AzureQueueStorage
+from ._generated.models import StorageErrorException, SignedIdentifier
+from ._generated.models import QueueMessage as GenQueueMessage
+
+from .models import QueueMessage, AccessPolicy, MessagesPaged
+
+if TYPE_CHECKING:
+    from datetime import datetime
+    from azure.core.pipeline.policies import HTTPPolicy
+    from .models import QueuePermissions, QueueProperties
+
+
+class QueueClient(StorageAccountHostsMixin):
+    """Creates a new QueueClient. This client represents interaction with a specific
+    queue, although that queue may not yet exist.
+
+    :param str queue_url: The full URI to the queue. This can also be a URL to the storage
+        account, in which case the queue must also be specified.
+    :param queue: The queue. If specified, this value will override
+        a queue value specified in the blob URL.
```

Comment:
blob?

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-python/pull/6186